src/repositories/redis/oidc-provider.ts

```diff
@@ -140,4 +172,6 @@ class RedisAdapter {
   }
 }
 
-export default RedisAdapter;
+export const oidcProviderAdapterFactory: AdapterFactory = (name) => {
```

Comment:
suggestion: 
- move oidcProviderAdapterFactory in `src/services/oidc-provider-adapter.ts`
- move PostgresAdapter in `src/repositories/oidc-provider-adapter-for-clients.ts` and rename to OidcProviderAdapterForClients
- move RedisAdapter in `src/repositories/redis/oidc-provider-adapter.ts` and rename to OidcProviderAdapter

Reply:
:ok: